withdraw/withdraw.go

```diff
@@ -31,6 +31,10 @@ func TxBlock(ctx context.Context, l2c *rpc.Client, l2TxHash common.Hash) (*big.I
 	return receipt.BlockNumber, nil
 }
 
+func ProofFinalized(ctx context.Context, portal *bindings.OptimismPortal, l2TxHash common.Hash) (bool, error) {
```

Comment:
we could alternatively just put this code in main.go, but it's nice to have here for other packages which might inherit this code